src/org/labkey/test/util/EscapeUtil.java

```diff
@@ -212,9 +212,22 @@ public static String getMarkupEscapedValue(String value)
     }
 
     private static final Pattern nameExpressionNeedsEscaping = Pattern.compile("([\\\\$/&}~,.])");
-    public static String escapeForNameExpression(String name)
+    public static String escapeForNameExpression(String value)
     {
-        return nameExpressionNeedsEscaping.matcher(name).replaceAll("\\\\$1");
+        return nameExpressionNeedsEscaping.matcher(value).replaceAll("\\\\$1");
+    }
+
+    private static final Pattern excelPageNeedsEscaping = Pattern.compile("([:/])");
+    /**
+     * Escapes invalid characters in a string to ensure it can be used as a valid Excel sheet name.
+     * Replaces characters matching the {@code excelPageNeedsEscaping} pattern with an underscore ("_").
+     *
+     * @param value the input string to be escaped
+     * @return the escaped string that can safely be used as an Excel sheet name
+     */
+    public static String escapeForExcelSheetName(String value)
+    {
+        return excelPageNeedsEscaping.matcher(value).replaceAll("_");
     }
```

Comment:
There's an existing, more thorough method for this (it's what is used on the server side). `org.apache.poi.ss.util.WorkbookUtil#createSafeSheetName(java.lang.String, char)`

```suggestion
    public static String createSafeSheetName(String value)
    {
        return WorkbookUtil.createSafeSheetName(value, '_');
    }
```

Also, this isn't escaping (escaping is reversible). Consider moving this method to `ExcelHelper`.

Reply:
Changed.